classes/fields/currency.php

```diff
@@ -285,6 +285,11 @@ public function validate( $value, $name = null, $options = null, $fields = null,
 	 */
 	public function pre_save( $value, $id = null, $name = null, $options = null, $fields = null, $pod = null, $params = null ) {
 
+		if ( $this->is_empty( $value ) ) {
+			// Don't enforce a default value here
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: eb40e2b8a7fb30232e0dde51e4509213</sub>